### ISSUE TITLE
Report file contains test name instead path

### ISSUE THREAD

**Issue by sartymlg** [tornaia/karma-sonarqube-unit-reporter#4]
Hi, thanks for develop this plugin. I'm trying to import my karma reports to sonar and seems that this is the best solution. However, i can't use it, because in my case, it generates this report

```xml
<unitTest version="1">
  <file path="Controller: MainCtrl">
    <testCase name="should attach a list of awesomeThings to the scope" duration="3"/>
  </file>
</unitTest>
```

Sonar generic import expects as real path in the path attribute, do you have the same problem?

**Comment by tornaia** [comment]
Yep, I also faced with that issue so I refactored a little bit the test codes to make them Sonar compatible.

Example:
project\test\app\dashboard\dashboard.controller.spec.js

```
(function() {
  'use strict';

  describe('app\\dashboard\\dashboard.controller.spec.js', function() {
    // ...

    beforeEach(inject(function(_$controller_, _applicationInfoService_, _moment_, _$rootScope_) {
      // ...
    }));

    it('should call applicationInfoService.getApplicationInfo once', function() {
      // ...
    });

  });
})();
```

In short: in the describe section I specify the path to the test file.
